Sources/Commands/SwiftTestTool.swift

```diff
@@ -71,9 +71,27 @@ struct SharedOptions: ParsableArguments {
     /// to choose from (usually in multiroot packages).
     @Option(help: .hidden)
     var testProduct: String?
+
+    /// Whether to enable support for XCTest.
+    @Flag(name: .customLong("xctest"),
+          inversion: .prefixedEnableDisable,
+          help: "Enable support for XCTest")
+    var enableXCTestSupport: Bool = true
+
+    /// Whether to enable support for swift-testing.
+    @Flag(name: .customLong("experimental-swift-testing"),
+          inversion: .prefixedEnableDisable,
+          help: "Enable experimental support for swift-testing")
+    var enableSwiftTestingLibrarySupport: Bool = false
```

Comment:
Can both of these be enabled at the same time?

Reply:
Yes, it is possible to use both at the same time. It will (at least currently) result in two separate builds occurring, but that's something we can look at refining (on Linux, at least) in the future.